modules/editor/webhook/WebhookControls.tsx

```diff
@@ -68,38 +61,34 @@ export function WebhookControls(props: WebhookControlsProps) {
     }
   }, [])
 
-  return useObserver(() => (
-    <Stack gap={12}>
-      <InputField
-        ref={inputRef}
-        id="webhook"
-        type="password"
-        label="Webhook URL"
-        placeholder="https://discord.com/api/webhooks/..."
-        error={form.subForm("target").field("url").error}
-        {...form.subForm("target").field("url").inputProps}
-      >
-        <PrimaryButton
-          disabled={!editorManager.target.exists}
-          onClick={handleSend}
+  return useObserver(() => {
+    let submitLabel = "Submit"
+    if (editorManager.messages.every(m => !m.reference)) submitLabel = "Send"
+    else if (editorManager.messages.every(m => m.reference))
+      submitLabel = "Edit"
```

Comment:
Probably best to use blocks for these as they wrap past the column limit now
```suggestion
    if (editorManager.messages.every(m => !m.reference)) {
      submitLabel = "Send"
    } else if (editorManager.messages.every(m => m.reference)) {
      submitLabel = "Edit"
    }
```